lib/chef/knife/ec2_server_create.rb

```diff
@@ -111,9 +111,9 @@ class Ec2ServerCreate < Knife
         :short => "-T T=V[,T=V,...]",
         :long => "--tags Tag=Value[,Tag=Value...]",
         :description => "The tags for this server. [DEPRECATED] Use --aws-tag instead.",
-        :proc => Proc.new { |v|
+        :proc => Proc.new { |tags|
           Chef::Log.warn("[DEPRECATED] --tags option is deprecated. Use --aws-tag option instead.")
-          v
+          tags.split(',')
```

Comment:
We deprecated this flag two years ago. We should probably just remove the option rather than try to fix it in any way.

Reply:
Hi @rhass, I think there is some mismatch, Please correct me if  I am wrong `chef-tag`  & `aws-tag` option is introduced and `--tags` option is deprecated in PR: https://github.com/chef/knife-ec2/pull/520/files#diff-e2993b8367db674f58a052036f00052aR113, It's added in Latest Release 0.18.0 (2018-04-05). 
Please let me know your thought about that.

Reply:
Ahh! I must have looked at the wrong cset. Sorry about the confusion.